upgrading/manual_upgrades.adoc

```diff
@@ -805,11 +805,11 @@ node2.example.com           Ready                      165d
 == Upgrading the Router
 
 If you have previously
-xref:../../install_config/router/index.adoc#install-config-router-overview[deployed a router], the
+xref:../install_config/router/index.adoc#install-config-router-overview[deployed a router], the
 router deployment configuration must be upgraded to apply updates contained in
 the router image. To upgrade your router without disrupting services, you must
 have previously deployed a
-xref:../../admin_guide/high_availability.adoc#configuring-a-highly-available-service[highly-available
+xref:../admin_guide/high_availability.adoc#configuring-a-highly-available-service[highly-available
```

Comment:
Hmm I don't think highly-available should be hyphenated, but it is in the admin topic too

Reply:
Oops you said that and I missed updating it, sry.